### DISCUSSION TITLE
Save data plugin in document

### DISCUSSION THREAD

**Discussion by tobiasBora** [xournalpp/xournalpp#6098] [Q&A] [answered]
A plugin might want to store data in a document or in a stroke, but I can't find how to do. Is this already possible with the current API?

**Comment by rolandlo** [comment, marked as answer]
The LoadHandler and SaveHandler are responsible for loading and saving data from/to xopp-files. There is no way for plugins to change what kind of information they load/save. Loading/saving other data would require a change in the file format, which would raise issues with backward compatibility.
